components/social_links.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import twitterIcon from '../images/icons/twitter.svg';
+import githubIcon from '../images/icons/github.svg';
+import dribbbleIcon from '../images/icons/dribbble.svg';
+
+const Icons = {
+  twitter: twitterIcon,
+  github: githubIcon,
+  dribbble: dribbbleIcon,
+};
+
+export default class SocialLinks extends React.Component {
```

Comment:
PropTypes is potentially used in several lines (one for each prop), while Component is tipically used just once. it's not about being more explicit, nor about neglecting anything. I think you're once again reading too much into stuff :stuck_out_tongue: